gke/README.md

```diff
@@ -77,16 +79,21 @@ and removals.
 10. Once the plan has been verified, run command: `terraform apply` And when
     prompted, confirm the deployment. Deployment time can vary but has
 typically taken approximately ten minutes.
-11. Once deployment is complete, add the new GKE cluster to your local
-    Kubernetes configuration via gcloud by running the following command:
+11. Once deployment is complete, Terraform will display the name of the bastion host and the name and IP of the cluster. You can now connect to your bastion host:
+
+`gcloud compute ssh <bastion-host-name> --project <project-name> --region <region-name>`
+
+The `--project` and `--region` flags can be omitted if `gcloud` the default values configured during initialization can be used. If you have previously used service account credentials, switch back to use your user account credentials before connecting to the bastion: `gcloud auth login`.
+
+You will need to connect to the bastion whenever you need to access the cluster or some of its components.
+
+12. On the bastion, initialize gcloud: `gcloud init`
```

Comment:
On running this command I am prompted to to use either the bastion's service account, or create a new one.  Can we add a line here to specific which one the user should select